1.10/cli/index.md

```diff
@@ -29,28 +30,30 @@ Available DC/OS commands:
 	service        	Manage DC/OS services
 	task           	Manage DC/OS tasks
 
-Get detailed command description with 'dcos <command> --help'.
+Get detailed command description with `dcos <command> --help`.
 ```
- 
-# Environment Variables
+
+# Environment variables
 
 These environment variables are supported by the DC/OS CLI and can be set dynamically.
 
 #### DCOS_CONFIG
-Specifies the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
+The path to a DC/OS configuration file. If you put the DC/OS configuration file in `/home/jdoe/config/dcos.toml`, you would set the variable with the command:
 
 ```bash
 export DCOS_CONFIG=/home/jdoe/config/dcos.toml
 ```
-    
+
+The `DCOS_CONFIG` variable is supported only before you run the first [`dcos cluster setup`](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-setup) command. `dcos cluster setup` copies the file into `<home-directory>/.dcos/clusters/<cluster_id>/dcos.toml`, after which the variable is ignored. 
+
```

Comment:
Here is how I think of DCOS_CONFIG and how it currently works / will disappear going forward:
```
Setup flow for new users of the 0.4.x CLI (this is for DC/OS 1.9):
$ export DCOS_CONFIG=<path/to/config> (optional)
  (default when not set is ~/.dcos/dcos.toml)
$ dcos config set core.dcos_url <url>
$ dcos config set core.ssl_verify [“true” | “false” | <path/to/ca_cert>]
$ dcos auth login
  (generates  / updates global config at $DCOS_CONFIG where all settings are set)

Setup flow for new users of the 0.5.x CLI (this is for DC/OS 1.10):
$ export DCOS_DIR=<path/to/config_dir> (optional)
  (default when not set is ~/.dcos)
$ dcos cluster setup <url>
  (generates / updates per cluster config under $DCOS_DIR/clusters/<cluster_id> where all settings are set)
  (sets newly set up cluster config as the active one)

Automatic update of config when currently using the 0.4.x CLI and updating to new 0.5.x CLI:
$ export DCOS_CONFIG=<path/to/config> (optional)
  (default when not set is ~/.dcos/dcos.toml)
$ dcos <subcommand>

$ … download and install new dcos 0.5.x binary …

$ export DCOS_DIR=<path/to/config_dir> (optional)
  (default when not set is ~/.dcos)
$ dcos <subcommand>
  (triggers conversion from old config structure to new)
  (moves global config from $DCOS_CONFIG to $DCOS_DIR/clusters/<cluster_id>/dcos.toml)
  (sets newly converted cluster config as the active one)
  (DCOS_CONFIG is no longer honored)

Preserving backwards compatibility:
1. Before `dcos cluster setup <url>` is called, follow the old flow semantics
   and allow users to change the config pointed to by $DCOS_CONFIG
   (print warning message saying these calls are deprecated and recommend using `dcos cluster setup`)
2. After `dcos cluster setup` is called (or automatic conversion has occurred),
   update the ‘active’ cluster’s config whenever old flow commands are called
   (print different warning message saying the calls are deprecated and cluster config state may now be corrupted)
```